src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -671,7 +671,10 @@ public void AddChild(UIElement child, int? index = null)
 
 			OnChildAdded(child);
 
-			InvalidateMeasure();
+			child.InvalidateMeasure();
+
+			// Arrange is required to unset the uno-unarranged CSS class
+			child.InvalidateArrange();
```

Comment:
That's odd..  an `InvalidateMeasure` should do it.

Reply:
It's the arranged size has not changed, the native arrange is skipped.